biometricks/src/main/java/com/handstandsam/biometricks/internal/BiometricksHelper.kt

```diff
@@ -2,15 +2,23 @@ package com.handstandsam.biometricks.internal
 
 import android.content.Context
 import android.content.pm.PackageManager
+import android.hardware.fingerprint.FingerprintManager
 import android.os.Build
+import android.security.keystore.KeyGenParameterSpec
+import android.security.keystore.KeyProperties
+import android.util.Log
+import androidx.annotation.RequiresPermission
 import androidx.biometric.BiometricManager
 import com.handstandsam.biometricks.Biometricks
+import java.security.InvalidAlgorithmParameterException
+import java.security.KeyStore
+import javax.crypto.KeyGenerator
 
 
 /**
  * Internal logic to compute the [Biometricks] type.
  */
-internal class BiometricksHelper(context: Context) {
+internal class BiometricksHelper(private val context: Context) {
```

Comment:
This can go poorly.  If someone sends us an `Activity`, we'd cause a memory leak here.